Admin Portal Navigation

rluberto opened the pull request:
In this PR I updated the admin portal and added the new navigation bar. Let me know if you want me to change anything.

![image](https://user-images.githubusercontent.com/21092717/101714930-85c19c00-3a68-11eb-91bd-48bc2ddc4212.png)

![image](https://user-images.githubusercontent.com/21092717/101714973-9bcf5c80-3a68-11eb-9ec6-3a92f60edcba.png)

![image](https://user-images.githubusercontent.com/21092717/101715003-aab60f00-3a68-11eb-8e67-c80f23f76b2f.png)

Commit:
navigation bar

Commit:
final navigation bar

Commit:
remove outline from hamburger button

tbatk reviewed (approved):
Looks awesome! Do you think you could have some sort of component that dims the background when you open the menu on mobile? It could be a z-index higher than the page, but not higher than the menu. Clicking it could close the nav menu. That way users don't click on the buttons in the background when the nav menu is open.

VishalAiely commented on components/Portal/Navigation/index.tsx:
```diff
@@ -0,0 +1,64 @@
+                            </a>
+                        </div>
+                    </div>
+                    <div className={style.navLowerText}>
+                        A Hack4Impact Chapter
+                    </div>
```
We forgot to remove this section from the design but this part can be erased

VishalAiely reviewed (approved):
The dashboard looks really good and follows the design really well

Commit:
add mobile menu overlay

rluberto commented:
I added the overlay for the mobile menu @trevormangrum and I replaced the bottom text with their slogan @VishalAiely . Let me know if you want to keep their slogan at the bottom or remove that text completely. Also clicking on the blacked-out area will close the menu.

![image](https://user-images.githubusercontent.com/21092717/101818731-48a1ec00-3af2-11eb-90ff-46d4ab8586a0.png)

VishalAiely commented:
I think the statement at the bottom looks good @rluberto
